Apps/W1/ShopifyConnector/app/Customer/Codeunits/ShpfyCustomerAPI.Codeunit.al

```diff
@@ -0,0 +1,454 @@
+/// <summary>
+/// Codeunit Shpfy Customer API (ID 30114).
+/// </summary>
+codeunit 30114 "Shpfy Customer API"
+{
+    Access = Internal;
+
```

Comment:
This is correct, because the permissions are already in the role of "D365 BASIC"
Previous we use the permission sets in an xml file, because the permission objects didn't exists.
With the XML file, we must also included these permissions, but now with the permission set objects is this not necessary anymore.

Reply:
Updated with commit: "Inderect Permissions"